node.js

```diff
@@ -231,11 +228,21 @@ async function lookup(message, callback) {
         PROTO_PATH,
         "Node"
       );
-      const user = await successorClient.lookupUserRemoteHelper({
-        id: userId
-      });
-      callback(null, user);
+      await successorClient.lookupUserRemoteHelper(
+        { id: userId },
+        (err, user) => {
```

Comment:
This callback is pretty confusing to me right now.  I would expect this  helper to have the same return type as lookupUser and for all this logic in the callback to occur after the completion of the async/await code

Reply:
I think I did this because I had problems with a syntax like
`const { err, user } = lookupUser(message.request.id);`
because lookupUser is a local function and returns types, RemoteHelper is a gRPC and returns a function. 

There are other examples like
`nPrimeSuccessor = await nodeQueriedClient.findSuccessorRemoteHelper( )`
that are working fine, however this is because the errror is always null. 
`  callback(null, nPrimeSuccessor);`
When I started returning error codes as well as the User data type, I had problems with that syntax and I did not know how to deal with it.

I guess we can open an issue about this because I was struggling for a while to get the gRPC working and I think that was the only way I could do it.